### DISCUSSION TITLE
[PSD6] Example 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#145] [Esercizi (A-L)]
Semplificare la seguente equazione booleana: $AB + BC + \bar B \bar D + A \bar C \bar D$

<details>
<summary>Versione precedente (avevo trascritto erroneamente i NOT) del post:</summary>
- Semplificare la seguente equazione booleana: $AB + BC + \overline {BD} + A \overline {CD}$
</details>

**Comment by Elia-Belli** [comment]
- Soluzione:
$AB + BC + \bar B \bar D + A \bar C \bar D$
$AB + BC + \bar B \bar D + AB\bar C \bar D + A \bar B \bar C \bar D$
$AB (1 + \bar C \bar D) + BC + \bar B \bar D (1+A \bar C)$
$AB + BC + \bar B \bar D$

- Soluzione versione precedente del post:
$AB + BC + \overline {BD} + A \overline {CD}$
$AB + BC + \bar B + \bar D + A (\bar C + \bar D)$
$AB + BC + \bar B + \bar D + A \bar C$
$(\bar B + B)(\bar B + A) + (\bar B + B)(\bar B + C)+\bar D + A \bar C$
$\bar B + A + C + \bar D$

**Reply by neoSnakex34** [reply]
corretto

**Comment by Emanuele-Dambrosio** [comment]
Non mi torna il tuo penultimo passaggio, in ogni caso lascio la soluzione a questo link.
https://www.boolean-algebra.com/?q=YWIrYmMre2JkfSthe2NkfQ==

**Comment by Elia-Belli** [comment]
@Emanuele-Dambrosio avevo ricopiato male il testo dell'esercizio, adesso torna grazie

**Comment by FeddyLix17** [comment]
@Elia-Belli 
l'esercizio sulle slide sta scritto letteralmente cosi "AB + BC +B’D’ + AC’D’", non va interpretato come AB+BC+B&#x0304;C&#x0304;+AC&#x0304;D&#x0304;?

**Comment by Elia-Belli** [comment]
@FedVlogger17 hai ragione sullla slide è diverso, io ho seguito il testo del post di matypist
Nel dubbio aggiungo anche l'altra soluzione

**Comment by matypist** [comment]
> @Elia-Belli l'esercizio sulle slide sta scritto letteralmente cosi "AB + BC +B’D’ + AC’D’", non va interpretato come AB+BC+B̄C̄+AC̄D̄?

Confermo, nel scrivere il post ho commesso una piccola svista nel ricopiarlo dalla slide, scusate per il disagio e grazie per la segnalazione

**Comment by FeddyLix17** [comment]
@Elia-Belli 
la soluzione del testo corretto è corretta, tranne per una piccola svista al secondo passaggio dove D&#x0304;D&#x0304; dovrebbe essere invece B&#x0304;D&#x0304;

**Comment by Elia-Belli** [comment]
Corretto, grazie
